source/clear-linux/reference/bundles/openssh-server.rst

```diff
@@ -110,9 +110,35 @@ To enable root login via SSH, perform the following steps:
    .. code-block:: bash
 
       mkdir /etc/ssh
+      
+#. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
```

Comment:
I suggest following this up with a command the reader can issue so it's consistent with the format of the other steps. (i.e. $EDITOR /etc/ssh/sshd_config)

Reply:
confused as i mad this a close to "Change default port" sectioin, there we are not saying what editor to use just that this need to be added to the file.
we seem to go between showing a editor and not, as the top of this doc did not ask for one editor vs a nother, and the section in quatiuon could append more then one line to the file if ran more then once.

if this is the recomned drection then we need to chang all docs to refrect this change.
if we are going to standerdize on one editor than what one to use?
i voite for nano but that is not a proper option.

Reply:
For the change default port section the command given, `systemctl edit sshd.socket` automatically opens up an editor using the default editor. 

I agree with you standardization on this topic would improve the quality of our documentation overall. 

To clarify, my suggestion was to actually use the command `$EDITOR /etc/ssh/sshd_config` since the $EDITOR environment variable will point to the default editor program. I've seen this used in other places in our documentation. This approach relieves us from having to pick an editor in all examples.

Reply:
tested with $EDITOR /etc/ssh/sshd_config 
well that doped my into **vi**..!! not ok, as others and I do not know how to get out. or how to use.
for now we need something other then  $EDITOR or something about seating this environment variable beforehand.